gherkin/ruby/lib/gherkin/gherkin_query.rb

```diff
@@ -0,0 +1,54 @@
+module Gherkin
+  class Query
+    def initialize
+      @ast_node_locations = {}
+    end
+
+    def update(message)
+      update_feature(message.gherkin_document.feature) if message.gherkin_document
+    end
+
+    def get_location(ast_node_id)
```

Comment:
are these methods "synched up" across all implementations. If they are that's cool. If not traditionally we would drop `get_` and `set_` prefixes in methods.

Reply:
We aim to have a consistent API between implementations, but I still think we should be idiomatic. Please change to `location`.

Reply:
done